### DISCUSSION TITLE
Source code edit? Insert <script> tag?

### DISCUSSION THREAD

**Discussion by cimenta** [codex-team/editor.js#2432] [General]
Hi

I am looking for an editor for a blog site and time to time the users need to insert  and later on also edit  <script> tag into the post. But in a way that the script actually works.

Does editor allow users to insert and save working <script> tag?

Thank you
Radek

**Comment by mendhak** [comment]
Did you get an answer to this? I tried the 'raw' block but it doesn't seem to accept script, which I am wondering, could be related to security?
